src/data/courses/lego/robotics.md

```diff
@@ -1,36 +1,35 @@
 ---
 title: Robotika tanfolyam
-society: LEGO
+society: LEGO Kör
```

Comment:
Ezt nem kell átírnod, mert ez csak egy id. Lásd itt: https://github.com/simonyiszk/tanfolyam-web/blob/master/src/data/societies.yaml

Reply:
Ezért failel el a CI.

Reply:
@izsoandras, amennyiben azt szeretnéd, hogy a 'LEGO' helyett, a 'LEGO kör' legyen, akkor a [itt](https://github.com/simonyiszk/tanfolyam-web/blob/master/src/data/societies.yaml) is írd át.